### PR TITLE
fix: ticket cancellation mail message change

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6499

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/api/helpers/system_mails.py [hunk 1]
```diff
@@ -241,7 +241,7 @@
         'subject': 'Your order for {event_name} has been cancelled ({invoice_id})',
         'message': (
             "Hello,"
-            "<br/>your order for {event_name} has been cancelled by the organizer."
+            "<br/>your order for {event_name} has been cancelled."
```
Comment (iamareebjamal) [inline app/api/helpers/system_mails.py:244]: Check the message in the issue carefully. Match character by character

**Comment by iamareebjamal** [comment]
@maze-runnar Seems like you don't read the entire issue carefully at all. The PR fixes just 1 out of several things mentioned in the issue. Read the issue carefully. Why do you act like beginners sometimes

File: app/api/helpers/system_mails.py [hunk 2]
```diff
@@ -241,10 +241,11 @@
         'subject': 'Your order for {event_name} has been cancelled ({invoice_id})',
         'message': (
             "Hello,"
-            "<br/>your order for {event_name} has been cancelled."
-            "<br/>Please contact the organizer for more info." + "{cancel_msg}"
-            "<br/>To manage orders please login to <a href='{frontend_url}' target='_blank'>{frontend_url}</a>"
-            "and visit \"My Tickets\"."
+            "<br/>your order {order_id} for {event_name} has been cancelled."
```
Comment (iamareebjamal) [inline app/api/helpers/system_mails.py:244]:
```suggestion
            "<br/>Your order {order_id} for {event_name} has been cancelled."
```

**Comment by maze-runnar** [comment]
@sachinchauhan2889 can you please test it.

**Pull request merged**